lib/models/liturgical-day-def.ts

```diff
@@ -225,39 +225,39 @@ export default class LiturgicalDayDef implements BaseLiturgicalDayDef {
 
   /**
    * Retrieve the Martyrology items from the inherited LiturgicalDay object.
-   * @param key The key of the LiturgicalDay to create.
+   * @param id ID of the LiturgicalDay to create.
    * @param input The definition of the liturgical day.
-   * @param fromCalendar
+   * @param fromCalendarId
    * @param previousDef
    * @private
    */
   #retrieveMartyrologyData(
-    key: Key,
+    id: Id,
     input: LiturgicalDayInput,
-    fromCalendar: Key,
+    fromCalendarId: Id,
     previousDef?: LiturgicalDayDef,
   ): MartyrologyItem[] {
     // Retrieve the Martyrology items from the inherited LiturgicalDay object,
     // or create a new empty list.
     const martyrology: MartyrologyItem[] = previousDef?.martyrology ?? [];
 
     // [1] Then, check if Martyrology data exists from the date definition;
-    // [2] if martyrology data do not exists, but an inherited LiturgicalDay exists: do nothing more;
-    // [3] if martyrology no not exists, and there is no inheritance: take the def key as the martyrology item key.
-    (input.martyrology ?? (previousDef && []) ?? [key]).forEach((id: string | MartyrologyItemRedefined) => {
-      const pointer = typeof id === 'string' ? { key: id } : id;
+    // [2] if martyrology data do not exist, but an inherited LiturgicalDay exists: do nothing more;
+    // [3] if martyrology no not exists, and there is no inheritance: take the def ID as the martyrology item ID.
+    (input.martyrology ?? (previousDef && []) ?? [id]).forEach((id2: string | MartyrologyItemRedefined) => {
+      const pointer = typeof id2 === 'string' ? { id: id2 } : id2;
```

Comment:
I prefer `martyrologyId` here, instead of `id2`:
```suggestion
    (input.martyrology ?? (previousDef && []) ?? [id]).forEach((martyrologyId: string | MartyrologyItemRedefined) => {
      const pointer = typeof martyrologyId === 'string' ? { id: martyrologyId } : martyrologyId;
```